anitya/app.py

```diff
@@ -112,13 +112,17 @@ def after_openid_login(resp):
     login was successful on the OpenID server.
     '''
     default = flask.url_for('index')
+    blacklist = APP.config['BLACKLISTED_USERS']
```

Comment:
> APP.config.get('BLACKLISTED_USERS', []), to make sure that it is backwards compatible.

No need since it's defined in the default_config file

> maintain this blacklist in the private repo.

That's a good idea

> Also, will this really stop people?

It is easy to use another openid identity but since the case we've had so far is not really a real spammer, I figure the work needed to go around this blacklist is greater than the annoyance (at least I hope so).